examples/Demos/caduceus.scn

```diff
@@ -1,84 +1,5 @@
-
-        <EulerImplicitSolver name="cg_odesolver" rayleighMass="1" rayleighStiffness="0.03" />
-        <CGLinearSolver name="linear_solver" iterations="20" tolerance="1e-12" threshold="1e-18" />
-	<MechanicalObject name="dofs"  scale="1" dy="2" position="@grid.position"  tags="NoPicking" />
-        <UniformMass totalMass="1.0" />
-        <HexahedronFEMForceField name="FEM" youngModulus="30000.0" poissonRatio="0.3" method="large" updateStiffnessMatrix="false" printLog="0" />
-
-	<UncoupledConstraintCorrection />
-
-	<Node name="Collis">
-            <MeshObjLoader name="loader" filename="mesh/meca_snake_900tri.obj" />
-            <Mesh src="@loader" />
-            <MechanicalObject src="@loader" name="CollisModel" />
-            <Triangle  selfCollision="0" />
-            <Line    selfCollision="0" />
-            <Point  selfCollision="0" />
-            <BarycentricMapping input="@.." output="@." />
-        </Node>
-
-        <Node name="VisuBody" tags="Visual" >
-            <OglModel  name="VisualBody" fileMesh="mesh/snake_body.obj"   />
-            <BarycentricMapping input="@.." output="@VisualBody" />
-        </Node>
-
-        <Node name="VisuCornea" tags="Visual" >
-            <OglModel  name="VisualCornea" fileMesh="mesh/snake_cornea.obj"   />
-            <BarycentricMapping input="@.." output="@VisualCornea" />
-        </Node>
-
-        <Node name="VisuEye" tags="Visual" >
-            <OglModel  name="VisualEye" fileMesh="mesh/snake_yellowEye.obj"   />
-            <BarycentricMapping input="@.." output="@VisualEye" />
-        </Node>
-    </Node>
-
-    <Node name="Base" >
-        <Node name="Stick">
-                 <MeshObjLoader name="loader" filename="mesh/collision_batons.obj" />
-                 <Mesh src="@loader" />
-                 <MechanicalObject src="@loader" name="CollisModel" />
-                 <Line simulated="false" moving="false" />
-                 <Point simulated="false"  moving="false"/>
-        </Node>
-        <Node name="Blobs">
-                 <MeshObjLoader name="loader" filename="mesh/collision_boules_V3.obj" />
-                 <Mesh src="@loader" />
-                 <MechanicalObject src="@loader" name="CollisModel" />
-                 <Triangle simulated="false" moving="false"/>
-                 <Line simulated="false" moving="false"/>
-                 <Point simulated="false" moving="false"/>
-        </Node>
-
-            <Node name="Foot">
-            <MeshObjLoader name="loader" filename="mesh/collision_pied.obj" />
-            <Mesh src="@loader" />
-            <MechanicalObject src="@loader" name="CollisModel" />
-            <Triangle simulated="false" moving="false"/>
-            <Line simulated="false" moving="false"/>
-            <Point simulated="false" moving="false"/>
-        </Node>
-
-        <Node name="Visu" tags="Visual" >
-            <OglModel  name="OglModel" fileMesh="mesh/SOFA_pod.obj" />
-        </Node>
-    </Node>
+<?xml version='1.0' ?>
```

Comment:
Hello Erwan...replacing caduceus  is probably a mistake :)

Reply:
Yeah I know. I did it for testing. (simply press ctrl+r in qtcreator then everything show up properly without modifying command line or reload scenes ... *lazy*)
Of course I have to modify this file, don´t worry ;)